src/transformers/processing_utils.py

```diff
@@ -993,6 +990,59 @@ def apply_chat_template(
         )
 
 
+def _check_reversed_images_text_for_vlms(images, text):
```

Comment:
This sounds like the images or text themselves are reverse (flipped) i.e. sdrawkcab. I'd suggest something like `_validate_images_text_input_order`